co-authors-plus.php

```diff
@@ -66,6 +67,12 @@ class CoAuthors_Plus {
 
 	var $to_be_filtered_caps = array();
 
+	/**
+	 * Block editor functionality in the sidebar is
+	 * disabled by default.
+	 */
+	const SIDEBAR_PLUGIN_ENABLED = false;
```

Comment:
What's the purpose of this constant?

Reply:
It is outdated! Good catch – I'll update it momentarily.